<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log_search</class>
 <widget class="QDialog" name="Log_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>搜索地点</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>搜索附近</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>两点寻路</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>多点寻路</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>切换沙河地图</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>切换跨校区地图</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>切换本部地图</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>设置时间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>设置速度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>前往教室</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>查看地点</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>模糊搜索</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>根据操作查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>根据时间查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_begin_w">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_begin_h">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_begin_m">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_end_m">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_end_h">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_end_w">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起始时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>终止时间</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>星期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>星期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
